redhat-appstudio/service-provider-integration-operator
# Tokenuploader send SPIAccessToken Name as a part of Secret instead of label

### What does this PR do?
It modifies Tokenuploader controller the way that SPIAccessToken Name is sent as a part of Secret instead of label


### What issues does this PR fix or reference?
It is a continuation of https://issues.redhat.com/browse/SVPI-382 since it was realized that label limits amount of symbols for resource name.



### How to test this PR?
make sure SPIOperator configured with ENABLETOKENUPLOAD=true (see Admin Guide for details)
Obtain Access Token ($AT) from the Service Provider of your choice. For example from GitHub->Settings->Developer Settings->Personal Access Tokens
Create Secret l: kubectl apply -f my.yaml
With my.yaml file content like:
`
apiVersion: v1
kind: Secret
metadata:
  name: upload-secret
  labels:
    spi.appstudio.redhat.com/upload-secret: token
type: Opaque
stringData:
  providerUrl: https://github.com
  spiTokenName: my-spi-access-token
  tokenData: $AT
`

Make sure SPIAccessToken is created and Ready state (may take some time): kubectl get spiaccesstoken my-spi-access-token -o yaml 
(status:phase:Ready)
